Casks/g/grisbi.rb

```diff
@@ -9,7 +9,7 @@
   homepage "https://www.grisbi.org/"
 
   livecheck do
-    url "https://sourceforge.net/projects/grisbi/files/grisbi%20stable/"
+    url "https://sourceforge.net/projects/grisbi/rss?path=/grisbi%20stable"
     regex(%r{url=.*?/grisbi/files/grisbi%20stable/.*?[-_/](\d+(?:[-.]\d+)+)\.dmg[-_/%.]}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/Grisbi[^"' >]*?[._-]v?(\d+(?:[.-]\d+)+)\.dmg}i)
```

We can improve this regex a bit while we're at it:

* We can omit the directories in the regex, since we're restricting the RSS feed path.
* We can also reduce the ending anchor to just `\.dmg`, as `[-_/%.]` doesn't seem necessary at this point.
* We can make the first part of the filename explicit and align it with typical patterns (i.e., `Grisbi[._-]`). The existing regex is presumably using `.*?` to be able to match the older `Grisbi-Aqua` prefix but the newer files all use a `Grisbi-1.2.3.dmg` format.
* In general, this suggestion also makes some tweaks to bring this in line with typical patterns.